app/templates/02_programming_overview.md

```diff
@@ -1,13 +1,17 @@
 <<[prev]({{int_running_code}}) [index]({{int_index}}) [next]({{int_first_steps}})>>
 # Programming is Simple
-Programming is both a broad and a deep field, however, getting your foot in the door shouldn't be intimidating.
+Programming is both a broad and a deep field; however, getting your foot in the door shouldn't be intimidating.
+
 Programming is the act of telling a computer how to do things.
-Any game that you play, computer tool that you use; All of these things that are and that could be require code somewhere in the toolchain.
-With programming, you take data in, manipulate it, and you put data back out. 
-In these 3 non-mandatory steps lie the keys to harnessing the power of a computer.
-These inputs can be key presses, text, or arbitrary data.
+
+Any game that you play, computer tool that you use, app that you run - all programs that are and that could be require code somewhere in the toolchain.
+
+With programming, you take data in, manipulate it, and put data back out. It is within these 3 non-mandatory(?) steps that lie the keys to harnessing the power of a computer.
+These inputs can be key presses, text, or even arbitrary data.
 The outputs could be text that you read, pretty (or not so pretty) graphics, more data in any shape or size.
-I hope to use Python to show you that programming is simple.
-There are building blocks that anyone can wrap their head around with a bit of explanation and practice.
-The trick lies in realizing that even complicated problems can be expressed with these basic building blocks.
-The tools programming languages provide that extend the basics exist to let you express and solve problems in simpler, faster, or more beautiful ways.
+
+I hope to use Python to show you that programming is simple. Anyone can learn it.
+With patience, practice, and simple explanation, even the most steadfast skeptic can understand the basic building blocks of programming.
+The trick lies in realizing that even complicated problems can be expressed through the utilization of these building blocks. Everything can be reduced back to the basics!
```

Comment:
...that even the most complicated problems can be expressed with these building blocks?
ah thats what i had before, any reason why the utilization bit is necessary?